go/controller/cluster.go

```diff
@@ -73,16 +73,21 @@ func (c Cluster) JobRunning(job *paddlejob.TrainingJob) (bool, error) {
 	return false, nil
 }
 
-// getPodsTotalRequestsAndLimits accumulate resource requests and limits from all pods containers.
+// getPodsTotalRequestsAndLimits accumulate resource requests and
+// limits from all pods containers.
 func getPodsTotalRequestsAndLimits(podList *v1.PodList) (reqs v1.ResourceList, limits v1.ResourceList, err error) {
 	reqs, limits = v1.ResourceList{}, v1.ResourceList{}
 	for _, pod := range podList.Items {
 		for _, container := range pod.Spec.Containers {
 			AddResourceList(reqs, container.Resources.Requests)
 			AddResourceList(limits, container.Resources.Limits)
 		}
+
+		for _, container := range pod.Spec.InitContainers {
```

Comment:
from ChenXi's article we are going to use the controller in a general cluster, so could be pods other than Paddle pods.

Reply:
You are right! Thanks!